testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -97,6 +101,87 @@ QUnit.testInActiveWindow('Form\'s inputs saves value on refresh', function(asser
     assert.deepEqual(formData, { name: 'test' }, 'value updates');
 });
 
+
+QUnit.module('(T977436)', () => {
```

Comment:
It will better to add comment instead of ticket number as a module name. Module should have full description that will reflect the meaning

Reply:
Module name is strange
imho tests should be in QUnit.module('Adaptivity');

Reply:
Yep fixed. Move those test to the Adaptivity module